Detectors/GlobalTracking/include/GlobalTracking/MatchITSTPCQC.h

```diff
@@ -63,69 +63,71 @@ class MatchITSTPCQC
-  TEfficiency* getFractionITSTPCmatchEta(matchType m) const { return mFractionITSTPCmatchEta[m]; }
-
-  TH2F* getHistoEtaVsPtNum(matchType m) const { return mEtaVsPtNum[m]; }
-  TH2F* getHistoEtaVsPtDen(matchType m) const { return mEtaVsPtDen[m]; }
-  TEfficiency* getFractionITSTPCmatchEtaVsPt(matchType m) const { return mFractionITSTPCmatchEtaVsPt[m]; }
-
-  TH2F* getHistoClsVsPtNum(matchType m) const { return mClsVsPtNum[m]; }
-  TH2F* getHistoClsVsPtDen(matchType m) const { return mClsVsPtDen[m]; }
-  TEfficiency* getFractionITSTPCmatchClsVsPt(matchType m) const { return mFractionITSTPCmatchClsVsPt[m]; }
-
-  TH2F* getHistoChi2VsPtNum(matchType m) const { return mChi2VsPtNum[m]; }
-  TH2F* getHistoChi2VsPtDen(matchType m) const { return mChi2VsPtDen[m]; }
-  TEfficiency* getFractionITSTPCmatchChi2VsPt(matchType m) const { return mFractionITSTPCmatchChi2VsPt[m]; }
-
-  TH1F* getHistoPtPhysPrimNum(matchType m) const { return mPtPhysPrimNum[m]; }
-  TH1F* getHistoPtPhysPrimDen(matchType m) const { return mPtPhysPrimDen[m]; }
-  TEfficiency* getFractionITSTPCmatchPhysPrim(matchType m) const { return mFractionITSTPCmatchPhysPrim[m]; }
-
-  TH1F* getHistoPhiPhysPrimNum(matchType m) const { return mPhiPhysPrimNum[m]; }
-  TH1F* getHistoPhiPhysPrimDen(matchType m) const { return mPhiPhysPrimDen[m]; }
-  TEfficiency* getFractionITSTPCmatchPhiPhysPrim(matchType m) const { return mFractionITSTPCmatchPhiPhysPrim[m]; }
-
-  TH1F* getHistoEtaPhysPrimNum(matchType m) const { return mEtaPhysPrimNum[m]; }
-  TH1F* getHistoEtaPhysPrimDen(matchType m) const { return mEtaPhysPrimDen[m]; }
-  TEfficiency* getFractionITSTPCmatchEtaPhysPrim(matchType m) const { return mFractionITSTPCmatchEtaPhysPrim[m]; }
-
-  TH2F* getHistoResidualPt() const { return mResidualPt; }
-  TH2F* getHistoResidualPhi() const { return mResidualPhi; }
-  TH2F* getHistoResidualEta() const { return mResidualEta; }
-
-  TH1F* getHistoChi2Matching() const { return mChi2Matching; }
-  TH1F* getHistoChi2Refit() const { return mChi2Refit; }
-  TH2F* getHistoTimeResVsPt() const { return mTimeResVsPt; }
-  TH1F* getHistoDCAr() const { return mDCAr; }
-  TH2F* getHistoDCArVsPtNum() const { return mDCArVsPtNum; }
-  TH2F* getHistoDCArVsPtDen() const { return mDCArVsPtDen; }
-  TEfficiency* getFractionITSTPCmatchDCArVsPt() const { return mFractionITSTPCmatchDCArVsPt; }
-
-  TH1D* getHisto1OverPtNum(matchType m) const { return m1OverPtNum[m]; }
-  TH1D* getHisto1OverPtDen(matchType m) const { return m1OverPtDen[m]; }
-  TEfficiency* getFractionITSTPCmatch1OverPt(matchType m) const { return mFractionITSTPCmatch1OverPt[m]; }
-
-  TH1D* getHisto1OverPtPhysPrimNum(matchType m) const { return m1OverPtPhysPrimNum[m]; }
-  TH1D* getHisto1OverPtPhysPrimDen(matchType m) const { return m1OverPtPhysPrimDen[m]; }
-  TEfficiency* getFractionITSTPCmatchPhysPrim1OverPt(matchType m) const { return mFractionITSTPCmatchPhysPrim1OverPt[m]; }
+  /* do we need these?
```

Comment:
what is the problem in keeping them?

Reply:
No problem at all just wondering if this is deadcode or if there is use case. Anyways I just commented them out to test if something will fail.